jina/docker/hubio.py

```diff
@@ -275,12 +280,17 @@ def _check_docker_image(self, name: str) -> None:
 
         self.logger.info(f'✅ {name} is a valid Jina Hub image, ready to publish')
 
-    def _docker_login(self) -> None:
+    def _docker_login(self, docker_username: str = None, docker_password: str = None) -> None:
         """A wrapper of docker login """
         from docker.errors import APIError
+        login_username = docker_username
+        login_password = docker_password
```

Comment:
You can remove the `login_*` variables and use the `docker_*` variables instead.

Reply:
using `login_*` variables to signify that they will be used for login explicitly. These would be `self.args.username` and `self.args.password` by default, if the `self.args` are `None`, we are using the injected arguments `docker_username` and `docker_password`. Named distinctly for distinguishing the conditions.